compositor_integration_tests/src/tests/schedule_update.rs

```diff
@@ -7,6 +7,11 @@ use crate::{
 use anyhow::Result;
 use serde_json::json;
 
+// We register an output stream with an initial scene containing a single input stream.
+// Immediately after, we update the output stream to display two input streams. The update is scheduled to happen after 2 seconds.
+//
+// Output dump should display `input_1` for 2 seconds and then transition to display both `input_1` and `input_2` alongside each other.
+// `input_1` and `input_2` should appear for 8 seconds.
```

Comment:
This is fine, but it is unnecessarily wordy. It should be
```suggestion
// Show `input_1` for 2 seconds.
// Show `input_1` and `input_2` side by side (transition with animation)
```